src/main/scala/chisel3/util/SRAM.scala

```diff
@@ -211,7 +303,15 @@ object SRAM {
     implicit evidence: T <:< Vec[_],
     sourceInfo:        SourceInfo
   ): SRAMInterface[T] =
-    masked_memInterface_impl(size, tpe)(numReadPorts, numWritePorts, numReadwritePorts, Builder.forcedClock, None)
+    memInterface_impl(size, tpe)(
+      Seq.fill(numReadPorts)(Builder.forcedClock),
+      Seq.fill(numWritePorts)(Builder.forcedClock),
+      Seq.fill(numReadwritePorts)(Builder.forcedClock),
+      None
+    )(
+      Some(evidence),
+      sourceInfo
+    )
```

Comment:
```suggestion
  ): SRAMInterface[T] = {
    val clock = Builder.forcedClock
    memInterface_impl(size, tpe)(
      Seq.fill(numReadPorts)(clock),
      Seq.fill(numWritePorts)(clock),
      Seq.fill(numReadwritePorts)(clock),
      None
    )(
      Some(evidence),
      sourceInfo
    )
  }
```
This would be cleaner with just assigning clock to a `val`, also that's just generally a good idea because `Builder.forcedClock` requires accessing the Chisel runtime in the Dynamic variable which is just unnecessary extra compute to do more than once here.

Reply:
Same suggestion for the other places calling `Builder.forcedClock` a bunch.